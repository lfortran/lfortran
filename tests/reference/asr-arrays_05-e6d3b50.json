{
    "basename": "asr-arrays_05-e6d3b50",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/arrays_05.f90",
    "infile_hash": "09eac1191b2e8a76fafedf1307b2d1240ffbc5f632c08afa847df805",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-arrays_05-e6d3b50.stdout",
    "stdout_hash": "604f4b476a34d884e0dca21ac7c63e36453b857e02838a18a99582fe",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}