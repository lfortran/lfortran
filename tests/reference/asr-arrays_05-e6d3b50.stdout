(TranslationUnit (SymbolTable 1 {arrays_05: (Program (SymbolTable 2 {i: (Variable 2 i Local () () Default (Integer 4 []) Source Public Required .false.), numbers: (Variable 2 numbers Local (ArrayConstant [(RealConstant 1.500000 (Real 4 [])) (RealConstant 3.200000 (Real 4 [])) (RealConstant 4.500000 (Real 4 [])) (RealConstant 0.900000 (Real 4 [])) (RealConstant 7.200000 (Real 4 []))] (Real 4 [((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 5 (Integer 4 [])))])) () Save (Real 4 [((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 5 (Integer 4 [])))]) Source Public Required .false.)}) arrays_05 [] [(DoLoop ((Var 2 i) (IntegerConstant 1 (Integer 4 [])) (IntegerConstant 5 (Integer 4 [])) ()) [(= (ArrayItem (Var 2 numbers) [(() (Var 2 i) ())] (Real 4 []) ()) (RealBinOp (Cast (Var 2 i) IntegerToReal (Real 4 []) ()) Mul (RealConstant 2.000000 (Real 4 [])) (Real 4 []) ()) ())]) (= (Var 2 numbers) (ArrayConstant [(RealConstant 1.500000 (Real 4 [])) (RealConstant 3.200000 (Real 4 [])) (RealConstant 4.500000 (Real 4 [])) (RealConstant 0.900000 (Real 4 [])) (RealConstant 7.200000 (Real 4 []))] (Real 4 [((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 5 (Integer 4 [])))])) ())])}) [])
